crud/common/call.lua

```diff
@@ -330,9 +303,8 @@ function call.any(vshard_router, func_name, func_args, opts)
     end
     local replicaset_id, replicaset = next(replicasets)
 
-    local res, err = retry_call_with_master_discovery(vshard_router, replicaset, 'call',
-        func_name, func_args, {timeout = timeout},
-    'read', {})
+    local res, err = call_with_retry_and_recovery(vshard_router, replicaset, 'call',
+        func_name, func_args, {timeout = timeout}, true)
```

Comment:
```suggestion
        func_name, func_args, {timeout = timeout}, false)
```
is_single_call should be false here, most of `call.any()` calls go to multiple (if not all) replicasets